src/components/events/create-event/FormContainer.js

```diff
@@ -109,10 +109,13 @@ const FormContainer = () => {
             photo: getBase64(photo),
           };
           axios
-            .post("http://localhost:5100/graphql", {
-              query: print(ADD_EVENT),
-              variables: { input: values },
-            })
+            .post(
+              "http://neighborhoodchef-be-prod.us-east-1.elasticbeanstalk.com/graphql",
```

Comment:
Should eventually refactor these explicit URL values to use some kind of global or environment variable instead.